taskcluster/kinds/shortlist/kind.yml

```diff
@@ -39,7 +40,7 @@ tasks:
                 - dependencies
                 - worker.env
                 - attributes
-        worker-type: b-cpu-largedisk
+        worker-type: b-cpu-xlargedisk
```

Comment:
question: What changed about this that requires more disk space? AFAICT the inputs have not changed in any way.

Reply:
It's still not clear if it's the reason of failing, but I removed constant compression/decompression so now it can take more disk but less CPU and run faster which should be a good trade-off.